include/at_notif.rst

```diff
@@ -0,0 +1,21 @@
+.. _at_notif_readme:
+
+AT command notifications
+###############################
+
+AT command notifications can be dispatched to registered modules. 
+Modules can register a callback function to receive AT command notifications as
+raw string.  register a callback function to receive AT-command notifications as
+raw string.  Multiple instances, which can be identified by pointers to
+contexts, are also Modules can de-register the callback function to stop
+receiving notifications.
+
```

Comment:
Sorry, looks like this was messed up by the suggestions. :(

Should be like this:

AT command notifications can be dispatched to registered modules. 
Modules can register a callback function to receive AT command notifications as raw string.  
Multiple instances, which can be identified by pointers to contexts, are also supported.
Modules can de-register the callback function to stop receiving notifications.

Reply:
Replaced by the paragraph above to the at_notif.rst in the last push.